packages/p2p/src/stores/general-store.js

```diff
@@ -397,9 +397,7 @@ export default class GeneralStore extends BaseStore {
                     const server_time = this.props.server_time.get();
                     const blocked_until_moment = toMoment(blocked_until);
 
-                    this.user_blocked_timeout = setTimeout(() => {
-                        this.setUserBlockedUntil(null);
-                    }, blocked_until_moment.diff(server_time));
+                    if (server_time.isAfter(blocked_until_moment)) this.setUserBlockedUntil(null);
```

Comment:
because setTimeout has a max delay of 24.8 days, any other value set that is bigger than 24.8 will not work. Hence why I opted to check using isAfter rather than using setTimeout with an if check